src/main/java/com/griddynamics/cloud/learning/web/dto/UserDto.java

```diff
@@ -0,0 +1,21 @@
+package com.griddynamics.cloud.learning.web.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
```

Comment:
Please add an id

Reply:
If I do this user will have to explicitly provide an id when he registers. First of all why user should care about what id he will be assigned after registration and why should he know about id at all?

Reply:
"If I do this user will have to explicitly provide an id when he registers." - Why do you think so?